tests/accordion/accordion.browser-test.jsx

```diff
@@ -0,0 +1,249 @@
+						'no sea takimata sanctus est Lorem ipsum dolor sit amet.'
+				},
+				{
+					id: '2',
+					summary: 'The second item',
+					details: (
+						<div>
+							<p>Lorem ipsum dolor sit amet,</p>
+							<p>consetetur sadipscing elitr,</p>
+							<p>
+								sed diam nonumy eirmod tempor invidunt ut labore et dolore magna
+								aliquyam erat,
+							</p>
+							<p>sed diam voluptua.</p>
+							<p>At vero eos et accusam et justo duo dolores et ea rebum.</p>
+							<p>Stet clita kasd gubergren,</p>
+							<p>no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>
+						</div>
+					)
+				},
+				{
+					id: '3',
+					summary: 'The third item',
+					details: (
+						<div>
+							<p>
+								Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
+								diam nonumy eirmod tempor invidunt ut labore et dolore magna
+								aliquyam erat, sed diam voluptua.
+							</p>
+							<p>
+								At vero eos et accusam et justo duo dolores et ea rebum. Stet
+								clita kasd gubergren, no sea takimata sanctus est Lorem ipsum
+								dolor sit amet.
+							</p>
+						</div>
+					)
+				}
+			]
+		};
+	}
+
+	menuDropdown (selectedItem) {
+		return (
+			<Dropdown
+				align="right"
+				id="ButtonGroupExampleDropdown"
+				assistiveText="More Options"
+				buttonVariant="icon"
+				buttonClassName="slds-shrink-none"
+				iconName="down"
+				iconVariant="border-filled"
+				onSelect={(option) => {
+					if (option.label === 'delete') {
+						this.setState((state) => ({
+							...state,
+							items: state.items.filter((item) => item.id !== selectedItem.id)
+						}));
+					} else if (console) {
+						console.log('onSelect', event, option);
+					}
+				}}
+				openOn="click"
+				options={[
+					{
+						label: 'delete',
+						value: 'A0'
+					},
+					{
+						label: 'redo',
+						value: 'B0'
+					},
+					{
+						label: 'activate',
+						value: 'C0'
+					}
+				]}
+				iconSize="x-small"
+			/>
+		);
+	}
+
+	togglePanel (id) {
+		this.setState((state) => ({
+			...state,
+			expandedPanels: {
+				...state.expandedPanels,
+				[id]: !state.expandedPanels[id]
+			}
+		}));
+	}
+
+	render () {
+		return (
+			<IconSettings iconPath="/assets/icons">
+				<Accordion id="base-example-accordion" openOn="click">
+					{this.state.items.map((item) => (
+						<Panel
+							expanded={!!this.state.expandedPanels[item.id]}
+							id={item.id}
+							panelContentActions={this.menuDropdown(item)}
+							key={item.id}
+							onTogglePanel={() => this.togglePanel(item.id)}
+							summary={item.summary}
+						>
+							{item.details}
+						</Panel>
+					))}
+				</Accordion>
+			</IconSettings>
+		);
+	}
+}
+
+AccordionExample.displayName = 'AccordionExampleComponent';
+AccordionExample.propTypes = propTypes;
+AccordionExample.defaultProps = defaultProps;
+
+/* Accordion rendering tests
+ */
+
+describe('Accordion', function () {
+	describe('Renders Accordion', () => {
```

Comment:
These could easily be snapshots tests--that is "is X in the markup what it should be". Even the mouse actions, since we don't have state these could probably be snapshot tests-- that is we don't need to test the wrapping state container, just that when `isOpen`is true then the panel is open, etc.